### PR TITLE
Fix/improve deployment of solution in cluster sample 

### PR DESCRIPTION
**Pull request by remibm**
issue https://github.com/ODMDev/odm-dsi-docker/issues/110

### CODE REVIEW

File: dsi-runtime/samples/cluster/solution_deploy.sh [hunk 1]
```diff
@@ -3,69 +3,57 @@
...
-CONTAINER1=`get_ip dsi-runtime-container1`
-CONTAINER2=`get_ip dsi-runtime-container2`
-CONTAINER3=`get_ip dsi-runtime-container3`
-INBOUND=`get_ip dsi-runtime-inbound`
+function solution_deploy {
+        setvar DSI_IP "$1"
+        setvar RUNTIME_PORT "$2"
+        echo deploying solution on "$DSI_IP:$RUNTIME_PORT"
+        docker-compose run --rm -v $SRC_DIR:/dropins dsi-runtime-container1 /dsi-cmd solutionManager deploy remote /dropins/simple_solution-0.0.esa \
+        --host=$DSI_IP --port=$RUNTIME_PORT $SOL_MANAGER_OPTS || echo "Deploy failed, solution could be already deployed on $DSI_IP"
```
Comment (orsinije) [inline dsi-runtime/samples/cluster/solution_deploy.sh]: I would avoid to give such hint, it should come from the CLI itself.

File: dsi-runtime/samples/cluster/solution_deploy.sh [hunk 2]
```diff
@@ -3,69 +3,57 @@
...
-echo "CONTAINER2=$CONTAINER2"
-echo "CONTAINER3=$CONTAINER3"
-echo "INBOUND=$INBOUND"
+function connectivity_deploy {
+        setvar DSI_IP "$1"
+        setvar INBOUND_PORT "$2"
+        echo deploying connectivity on "$DSI_IP:$INBOUND_PORT"
+        docker-compose run --rm -v $SRC_DIR:/dropins dsi-runtime-container1 /dsi-cmd connectivityManager deploy remote /dropins/simple_solution-0.0.esa \
+        /dropins/in-connectivity-server-configuration.xml --server=dsi-runtime-inbound --host=$DSI_IP --port=$INBOUND_PORT $SOL_MANAGER_OPTS  \
+        || echo "Deploy failed, connectivity could be already deployed on $DSI_IP"
```
Comment (orsinije) [inline dsi-runtime/samples/cluster/solution_deploy.sh]: I would avoid to give such hint, it should come from the CLI itself.

**Comment by orsinije** [comment]
Sounds good, just a minor comment about 2 messages

File: dsi-runtime/samples/cluster/solution_deploy.sh (hunk 2 above)
Comment (remibm) [reply dsi-runtime/samples/cluster/solution_deploy.sh]: agree

File: dsi-runtime/samples/cluster/solution_deploy.sh (hunk 1 above)
Comment (remibm) [reply dsi-runtime/samples/cluster/solution_deploy.sh]: agree

**Pull request merged**